index.html

```diff
@@ -9,51 +9,235 @@
+            </ul>
+    
+            <button type="button">See Project</button>
+        </div>
+    </section>
+
+    <!--About Section-->
+    <section class="about">
+        <div class="about-me">
+            <h3 class="intro">About <br>Myself</h3>
+            <p class="main-paragraph">Hello, I'm a software developer! I can help you build a product, feature or website. 
+                Look through some of my work and experience! If you like what you see and have a project you need coded,
+                 don't hesitate to contact me.
+            </p>
+            <p class="connect">LET'S CONNECT</p>
+        </div>
+        
+        <div class="headline-section">
+            <ul>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/twiter.png" alt="Twitter"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social linkendIn">
+                        <img src="./images/Linkedinicon.png" alt="LinkendIn"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social twiter">
+                        <img src="./images/mail.png" alt="Email"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social github">
+                        <img src="./images/Vector.png" alt="GitHub"/>
+                    </a>
+                </li>
+                <li>
+                    <a href="#" class="social peace">
+                        <img src="./images/fingers.png" alt="Peace"/>
+                    </a>
+                </li>
+            </ul>
+        </div>
+        <button type="button" class="get-resume">Get my resume</button>
+
+        <!--Languages-Skills Section-->
+        <div class="skills-lang">
+            <ul>
+                <li id="languages">
+                    <a href="#" class="languages">
+                        <span class="lang-head">Languages</span> <img src="./images/Chevron-Down.png" alt="Chevron-Down"/>
+                    </a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/javaScript-icon.png" alt="JavaScript"/>
+                    <a href="#" class="lang javascript">JavaScript</a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/html-icon.png" alt="Html"/>
+                    <a href="#" class="lang html">HTML</a> 
+                </li>
+                <li class="about-lang">
+                    <img src="./images/css-icon.png" alt="CSS"/>
+                    <a href="#" class="lang css">CSS</a> 
+                </li>
+                <li class="framework-skills">
+                    <a href="#" class="framework">Framework</a>
+                    <img src="./images/Chevron-Right.png" alt="Right-caret"/>
```

Comment:
- [ ] At this part of your project, it would be better if you add more `padding` and change `text-color` on your different languages to match the Figma design.
![image](https://user-images.githubusercontent.com/80612925/170782894-dded9218-69f7-4bcd-85d8-b5c05f8cdea4.png)
2.**Figma design**
![image](https://user-images.githubusercontent.com/80612925/170782943-39860af4-4653-4830-900a-6670783d3bf3.png)